### PR TITLE
implemented solution

### PR DESCRIPTION
**Pull request by willwildred**
- [DEMO LINK](https://willwildred.github.io/layout_catalog/)
- [TEST REPORT LINK](https://willwildred.github.io/layout_catalog/report/html_report/)

- [x] All component follow BEM and use SCSS
- [x] repaeted sizes and special colors are put to variables
- [x] Grid is used for the columns
- [x] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [x] All changes on `:hover` are smooth
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,419 @@
...
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/src/styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
```
Comment (volodymyr-soltys97) [inline src/index.html:13]: Comment is redundant here
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,419 @@
...
+      </a>
+
+      <nav class="header__nav">
+        <ul class="nav-menu">
+
+          <li>
+            <a href="#" class="nav-menu__links is-active nav-menu__links--is-active">
+              Apple
+            </a>
+          </li>
```
Comment (volodymyr-soltys97) [inline src/index.html:26]: Fix classes everywhere
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
```

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,419 @@
...
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="/src/images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="nav-menu">
```
Comment (volodymyr-soltys97) [inline src/index.html:20]:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```

File: src/index.html [hunk 4]
```diff
@@ -1,12 +1,419 @@
...
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+            href="#"
+            class="nav-menu__links"
+            data-qa="nav-hover"
+            >
```
Comment (volodymyr-soltys97) [inline src/index.html:45]:
```suggestion
            <a
              href="#"
              class="nav-menu__links"
              data-qa="nav-hover"
            >
```

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,419 @@
...
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:207]: It will be better
```suggestion
      <article class="card">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    &-sum {
+      color: $text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    width: 166px;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:77]:
```suggestion
    width: 100%;
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done: 
Screenshot 2023-12-18 at 17 00 30
Screenshot 2023-12-18 at 17 00 38
Consider fixing font styles here

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**